addons/l10n_dk/data/account.account.template.csv

```diff
@@ -1,209 +1,209 @@
-"a1020","Salg af vare, EU","1020","income","account_tag_revenue","dk_chart_template","False"
-"a1021","Salg af ydelser, EU","1021","income","account_tag_revenue","dk_chart_template","False"
-"a1030","Salg af vare, 3. lande","1030","income","account_tag_revenue","dk_chart_template","False"
-"a1031","Salg af ydelser, 3. lande","1031","income","account_tag_revenue","dk_chart_template","False"
-"a2010","Direkte omkostninger vare, m/moms","2010","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2011","Direkte omkostninger ydelser, m/moms","2011","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2012","Direkte omkostninger vare, u/moms","2012","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2013","Direkte omkostninger ydelser, u/moms","2013","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2020","Direkte omkostninger vare, EU","2020","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2021","Direkte omkostninger ydelser, EU","2021","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2030","Direkte omkostninger vare, 3. lande","2030","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2031","Direkte omkostninger ydelser, 3. lande","2031","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2100","Vareforbrug, lagermodul","2100","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2110","Prisdifferencer","2110","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2600","Lager, primo","2600","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2605","Lager, ultimo","2605","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a2610","Lagerændring","2610","expense_direct_cost","account_tag_direct_costs","dk_chart_template","False"
-"a3010","Produktionsløn","3010","expense","account_tag_staff_costs","dk_chart_template","False"
+"a1010","Sales of goods, incl. VAT","1010","income","account_tag_revenue","dk_chart_template","False"
+"a1011","Sales of services, incl. VAT","1011","income","account_tag_revenue","dk_chart_template","False"
+"a1012","Sales of goods, excl. VAT","1012","income","account_tag_revenue","dk_chart_template","False"
+"a1013","Sales of services, excl. VAT","1013","income","account_tag_revenue","dk_chart_template","False"
+"a1020","Sale of goods, EU","1020","income","account_tag_revenue","dk_chart_template","False"
+"a1021","Sale of services, EU","1021","income","account_tag_revenue","dk_chart_template","False"
+"a1030","Sales of goods, 3. countries","1030","income","account_tag_revenue","dk_chart_template","False"
+"a1031","Sales of services, 3rd countries","1031","income","account_tag_revenue","dk_chart_template","False"
```

Comment:
You used 3. one time, and 3rd the second. Stay consistent throughout the loca :)